### ISSUE TITLE
GUI can't render script

### ISSUE THREAD

**Issue by escape0707** [sahib/rmlint#621]
I'm using Arch, BTRFS, and latest stable rmlint-shredder.

With default config, I use the GUI to scan my Pictures directory, and while it gives some dupe files, after hitting render script from all, the GUI just get stuck. The icon in the left panel is only circling, and I can't take any further steps.

I don't see any suspicious log when launching from command line with the --gui option. The program stuck after running the generated rmlint command. CLI works fine, but I need the GUI tool to verify and select the exact directory I don't need.

**Comment by Cebtenzzre** [comment]
I was able to reproduce this myself, and as far as I can tell it was caused by a [performance optimization](https://github.com/GNOME/glib/commit/71a7603e7ffab00f00416e8f90083a7cc61edd27) in glib 2.75.3 that caused a problematic cast in the logging callback to actually result in a segfault instead of a compiler warning. This affected command-line rmlint as well but only if you were using --no-with-color or sending its output to a pipe. Try building from latest master.
